.eslintrc.js

```diff
@@ -79,6 +79,25 @@ module.exports = {
         extensions: ['.js', '.jsx'],
       },
     ],
+    'react/jsx-no-bind': [
+      error,
+      {
+        ignoreDOMComponents: true,
+        ignoreRefs: true,
+        allowArrowFunctions: true,
+        allowFunctions: true,
+        allowBind: true,
```

Comment:
Is there anything left that it still disallows?

BTW this rule is about performance so we probably shouldn't shut it up like this